src/multiproof.rs

```diff
@@ -37,6 +37,9 @@ impl CRS {
         CRS { n, G, Q }
     }
 
+    /*
+     Evaluates the polynomial at a specific point.
```

Comment:
A commitment to a polynomial is different from evaluating that polynomial, so this comment would not be correct

Reply:
Sure thing. What would be the correct wording here?

Reply:
Sorry about the late reply. Comment can say "Commits to a polynomial in lagrange form"

Reply:
OK, addressed.